mcs/class/Makefile

```diff
@@ -105,6 +105,8 @@ monodroid_dirs := \
 	$(mobile_common_dirs)	\
 	Mono.CompilerServices.SymbolWriter	\
 	Mono.Btls.Interface	\
+	System.Runtime.Serialization.Formatters.Soap	\
+	System.Runtime.Remoting	\
```

Comment:
Just to wrap up: this change is fine. It won't implicitly alter the assemblies included in Xamarin.Android.